index.js

```diff
@@ -71,8 +77,8 @@ module.exports = function(fc) {
   }
 };
 
-function pluckCoods(multi) {
-  return multi.map(function(geom) {
-    return geom.coordinates;
-  });
+function pluckCoords(multi) {
+  return multi.reduce(function(memo, coords) {
+    return memo.concat(coords);
+  }, []);
```

Comment:
FYI you're creating a new array copy on every concat, and an unnecessary function binding on every `pluckCoords` call. It's much faster and better to use a simple loop here.

```js
var coords = [];
for (var i = 0; i < multi.length; i++) {
    for (var j = 0; j < multi[i].length; j++) coords.push(multi[i][j]);
}
```